ft/ft/test_command.py

```diff
@@ -4,7 +4,7 @@
 
 def test_add_dynamic_type():
     assert add_dynamic_type("True").fttype == T_BOOL
-    assert add_dynamic_type("-1223").fttype == T_INT
+    assert add_dynamic_type("-1223").fttype == T_STRING
```

Comment:
This is failing. I am changing it to pass. Probably it is a bug for now?

Reply:
I have just committed a fix to master such that this can be reverted to `T_INT`. If you rebase your branch, it should pass.